app/_locales/en/messages.json

```diff
@@ -663,7 +663,7 @@
     "message": "Medium"
   },
   "editGasPriceTooltip": {
-    "message": "This network uses an older version of gas pricing which requires a “Gas price” field. Gas price is the maximum amount you are willing to pay per unit of gas."
+    "message": "This network requires a “Gas price” field when submitting a transaction. Gas price is the maximum amount you are willing to pay per unit of gas."
```

Comment:
actually it should be Gas Price is the amount *you will pay* per unit of gas

Reply:
commented on the document too

Reply:
Updated!